app/views/trainees/start_statuses/edit.html.erb

```diff
@@ -4,6 +4,7 @@
   <% if @trainee_start_status_form.next_step_context? %>
     <%= render GovukComponent::BackLinkComponent.new(text: t(:back),
                                                      href: trainee_start_date_verification_path(@trainee, context: @trainee_start_status_form.context)) %>
+
```

Comment:
I don't see any changes to this file other than a newline.

Reply:
fixed